.rubocop.yml

```diff
@@ -27,6 +27,15 @@ Documentation:
 Metrics/ClassLength:
   Max: 150
 
+Metrics/MethodLength:
+  Exclude:
+    - app/services/json_species.rb
+
+Metrics/AbcSize:
+  Exclude:
+    - app/services/json_species.rb
```

Comment:
Maybe we can group by the attributes by category

```ruby
def attributes
   basic.merge(abilities).merge(skills)
end

private

def basic
 {
 }
end

def abilities
 {
 }
end

# And so on..
```

But I'm not sure is worth it and affects readability, let's sink on this and get back to it 🤓